gettsim/data/e_st_abzuege.yaml

```diff
@@ -1,56 +1,70 @@
-kifreib:
+kifreib_s_exm:
   name:
-    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes sowie Betreuungs- und Erziehungs- oder Ausbildungsbedarf)
+    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes).
     en:
   description:
     de: § 32(6) EStG
     en:
   values:
     1984-01-01:
-      value: 221
-      note:
+      value: 110
+      note: Wert in Euro. Der Betrag laut Gesetz ist 216 DM.
```

Comment:
I find this in value of 216 DM in [B. v. 24.01.1984 BGBl. I S. 113](https://offenegesetze.de/veroeffentlichung/bgbl1/1984/5#page=1). That is just Bekanntmachung of EStG 1983. Not sure if this is for 1983 or 1984 or if this value is completely new.